martin/src/cog/image.rs

```diff
@@ -14,21 +16,158 @@ use crate::{MartinResult, TileData};
 pub struct Image {
     /// The Image File Directory index represents IDF entry with the image pointers to the actual image data.
     ifd_index: usize,
+    /// The extent of the image in model units, represented as [`min_x`, `min_y`, `max_x`, `max_y`].
+    extent: [f64; 4],
+    /// The origin of the image in model units.
+    origin: [f64; 3],
     /// Number of tiles in a row of this image
     tiles_across: u32,
-    ///  Number of tiles in a column of this image
+    /// Number of tiles in a column of this image
     tiles_down: u32,
+    /// Tile size in pixels
+    tile_size: (u32, u32),
+    /// Resolution of the image in model units per pixel
+    resolution: (f64, f64),
```

Comment:
if I read "resolution", I would not expect this unit.
Let's be more explicit and name it


```suggestion
    model_units_per_pixel: (f64, f64),
```